shared/lib/entity-mocks.ts

```diff
@@ -64,6 +84,20 @@ export const createSection = async (
     order: 1,
   };
 
+  // Create indicators if they don't exist in the database
+  if (additionalData !== undefined) {
+    const baseWidgets = additionalData.baseWidgets;
+    if (Array.isArray(baseWidgets) === true) {
+      for (let widgetIdx = 0; widgetIdx < baseWidgets.length; widgetIdx++) {
```

Comment:
Each time you don't use a functional approach, something inside me dies

Reply:
You are not the only person working in this company, where we use some standards, always based in the context.

If you like to do it your way, be my guest